scripts/apps/monitoring/services/CardsService.ts

```diff
@@ -7,8 +7,24 @@ import {
     SENT_OUTPUT,
     SCHEDULED_OUTPUT,
 } from 'apps/desks/constants';
-import {appConfig} from 'appConfig';
-import {IMonitoringFilter} from 'superdesk-api';
+import {appConfig, extensions} from 'appConfig';
+import {IMonitoringFilter, IPersonalSpaceSection} from 'superdesk-api';
+
+export function getExtensionSections() {
```

Comment:
`getSections` isn't returning a promise, so you should not be using `Promise.all` and make this function return an array instead.